tests/test-rest-users-controller.php

```diff
@@ -674,12 +674,17 @@ public function test_delete_user_invalid_reassign_id() {
 		$this->assertErrorResponse( 'rest_user_invalid_reassign', $response, 400 );
 	}
 
-	public function test_get_item_schema() {
-		$request = new WP_REST_Request( 'GET', '/wp/v2/users/schema' );
+	public function test_options_request() {
+
+		$request = new WP_REST_Request( 'OPTIONS', '/wp/v2/users' );
```

Comment:
Weird, though I added a comment. Yes, though not quite the same, I'd expect to get the `get` / `update` / `delete` request arguments, as opposed to `get` / `create`.